developer/images/dependencies-update/hack/bin/is-pr-skipped.sh

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+usage() {
+    echo "
+Usage:
+    ${0##*/} [options]
+
+Check if a PR is required to submit commited changes.
+The environment variable GITHUB_TOKEN must be set with a token that
+allows the PRs in the repository to be read.
+
+Mandatory arguments:
+    -b, --target_branch
```

Comment:
Before running `run` function can we pre-check the mandatory arguments

Reply:
Fixed